### PR TITLE
BPMSPL-114 -tests commit (expiration date test, comments test)

### PR DESCRIPTION
**Pull request by jakubschwan**
Added 2 tests for verification functions
test for expiration date - OK
test for comment - FAIL

### CODE REVIEW

File: jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskCommentTest.java [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+            assertEquals(1, tasks.size());
+            TaskSummary taskSum = tasks.get(0);
+
+            Comment comment = TaskModelProvider.getFactory().newComment();
+            Date date = new Date();
+            ((InternalComment)comment).setAddedAt(date);
+            User user = TaskModelProvider.getFactory().newUser();
+            ((InternalOrganizationalEntity) user).setId("Troll");
+            ((InternalComment)comment).setAddedBy(user);
+            ((InternalComment)comment).setText(txt);
```
Comment (triceo) [inline jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskCommentTest.java]: Don't repeat yourself, have the cast happen only once - possibly at line 86.

File: jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskCommentTest.java [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            assertEquals(date, commentById.getAddedAt());
+            assertEquals(user, commentById.getAddedBy());
+            assertEquals(txt, commentById.getText());
+
+            Comment comment2 = TaskModelProvider.getFactory().newComment();
+            ((InternalComment)comment2).setAddedAt(new Date());
+            User user2 = TaskModelProvider.getFactory().newUser();
+            ((InternalOrganizationalEntity) user2).setId("Master");
+            ((InternalComment)comment2).setAddedBy(user2);
+            ((InternalComment)comment2).setText(txt+"asdf");
```
Comment (triceo) [inline jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskCommentTest.java]: Ditto.

File: jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskExpirationDateTest.java [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            assertEquals(1, tasks.size());
+            TaskSummary taskSum = tasks.get(0);
+
+            Date exDate = new Date();
+
+            assertNull(taskService.getExpirationDate(taskSum.getId()));
+
+            taskService.setExpirationDate(taskSum.getId(), exDate);
+            Date date = taskService.getExpirationDate(taskSum.getId());
+            assertNotNull(date);
```
Comment (triceo) [inline jbpm-human-task/jbpm-human-task-core/src/test/java/org/jbpm/services/task/TaskExpirationDateTest.java]: Given that ```exDate``` is never null and the ```assertEquals()```below, this assert is not necessary.

**Comment by mswiderski** [comment]
test merged and issue fixed, thanks @jakubschwan

**Pull request closed without merging**